doc/intro.md

```diff
@@ -39,5 +39,5 @@ If you'd like to define someone of them, their keys are: `:bcrypt :scrypt :pbkdf
                                        :iterations 100000}}
 ```
 
-The module has `make` (it generate a hashed password) and `check` (it verify encripted password with current string) functions to deal hashing process.
+The module has `make` (it generate a hashed password) and `check` (it verify encrypted password against current string) functions to deal with hashing process.
```

Comment:
''... verifies the encrypted password against the current ..."

Reply:
Good catch :pray: